storage/pkg/flagset/frontend.go

```diff
@@ -56,29 +56,22 @@ func FrontendWithConfig(cfg *config.Config) []cli.Flag {
 			Name:        "network",
 			Value:       "tcp",
 			Usage:       "Network to use for the storage service, can be 'tcp', 'udp' or 'unix'",
-			EnvVars:     []string{"STORAGE_FRONTEND_NETWORK"},
-			Destination: &cfg.Reva.Frontend.Network,
-		},
-		&cli.StringFlag{
-			Name:        "protocol",
-			Value:       "http",
-			Usage:       "protocol for storage service, can be 'http' or 'grpc'",
-			EnvVars:     []string{"STORAGE_FRONTEND_PROTOCOL"},
-			Destination: &cfg.Reva.Frontend.Protocol,
+			EnvVars:     []string{"STORAGE_FRONTEND_HTTP_NETWORK"},
+			Destination: &cfg.Reva.Frontend.HTTPNetwork,
 		},
 		&cli.StringFlag{
 			Name:        "addr",
 			Value:       "0.0.0.0:9140",
 			Usage:       "Address to bind storage service",
-			EnvVars:     []string{"STORAGE_FRONTEND_ADDR"},
-			Destination: &cfg.Reva.Frontend.Addr,
+			EnvVars:     []string{"STORAGE_FRONTEND_HTTP_ADDR"},
+			Destination: &cfg.Reva.Frontend.HTTPAddr,
 		},
 		&cli.StringFlag{
-			Name:        "url",
+			Name:        "public-url",
 			Value:       "https://localhost:9200",
 			Usage:       "URL to use for the storage service",
-			EnvVars:     []string{"STORAGE_FRONTEND_URL"},
-			Destination: &cfg.Reva.Frontend.URL,
+			EnvVars:     []string{"STORAGE_FRONTEND_PUBLIC_URL"},
```

Comment:
need change in drone https://github.com/owncloud/ocis/blob/d422a91a01b98893d0a10a7555092342755ee33b/.drone.star#L561

Reply:
Done.